source/docs/installation.md

```diff
@@ -26,7 +26,7 @@ If you run into dependency conflicts when trying to install Jigsaw globally, you
 Create a folder for your site:
 
 ```
-$ mkdir my-site && cd my-site
+$ mkdir my-site && cd "$_"
```

Comment:
This is less clear than just duplicating `my-site`, so we'll keep it the old way--thanks!